### ISSUE TITLE
[TTS Extended] Samsung Spica Freeze

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#49]
```
What steps will reproduce the problem?
1. Just open the settings
2.
3.

What is the expected output? What do you see instead?
Phone freezes

What version of the product are you using? On what operating system?
TTS: latest from market
OS: Android 2.1 (Samdroid version) on Samsung Spica (GT-I5700)

Please provide any additional information below.


```

Original issue reported on code.google.com by `jrofurt...@gmail.com` on 20 Sep 2010 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
it doesn't matter what version of TTS Extended you're using, it freezes on ALL 
of them! It works like a charm with Android 1.5 and doesn't work at all with 
A2.1. What's the anwser?
```

Original comment by `robert.r...@gmail.com` on 9 Oct 2010 at 5:19

**Comment by GoogleCodeExporter** [comment]
```
Sometimes TTS- Extendend  window shows up, but there is only Pico-TTS engine to 
select, and then phone freezes.
```

Original comment by `malywilc...@gmail.com` on 29 Oct 2010 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
Я купил русский голос для движка SVOX, но 
пользоваться им не могу по описаным выше 
причинам. Автор собирается как-либо 
исправлять ситуацию или мне требовать 
возврат денег? Спасибо.
```

Original comment by `jungleof...@gmail.com` on 29 Oct 2010 at 4:23

**Comment by GoogleCodeExporter** [comment]
```
as soon as i open TTS service extended the app crashes and freezes the Phone. 
using a galaxy spica with 2.1 firmware, samdroid 1.3.1 and leshak kernel 2.08.
```

Original comment by `kbk...@gmail.com` on 10 Nov 2010 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
Is something going on with this issue? There're several Spica users, most of 
them already on Android 2.1. And unfortunatelly, tts extended is the only one 
tts engine to use, at least for me. I need tts extended because it's the only 
one engine that works with SVOX Mobile Voice Polish, which I use. So I strongly 
ask You, at least to anwser this topic to let us know what's going on with this 
issue, and if we can expect any solution in future...
```

Original comment by `robert.r...@gmail.com` on 29 Nov 2010 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Same thing happens for me, everything is installed, even extra languages I have 
to pay for.. the TTS settings hangs the phone. If it's Extended TTS or Svox I 
can't tell, but pico works great, sadly it doesn't have localisation.
```

Original comment by `backanda@gmail.com` on 10 Dec 2010 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately, this is a known issue with TTS Extended on some devices.  The 
only fix at this time is to use platform TTS, included in Android 2.2 or later.
```

Original comment by `caseybur...@google.com` on 3 Jun 2011 at 5:57
- Changed title: **[TTS Extended] Samsung Spica Freeze**
- Changed state: **WontFix**
